docs/build/configure/_index.md

```diff
@@ -13,32 +13,34 @@ aliases:
+- [Modules](/build/configure/#modules): {{< glossary_tooltip term_id="module" text="Modules" >}} provide {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}}, which are a way to add resource types or models that are not built into Viam.
+- [Remote parts](/build/configure/#remote-parts): Remotes are a way to connect two separate machines so one can access the resources of the other.
+- [Sub-parts](/build/configure/#sub-parts): Sub-parts are a way to connect two computers inside the same machine.
+- [Fragments](/build/configure/#fragments): Fragments are a way of sharing and managing identical configuration files (or parts of config files) across multiple machines.
+- [Frames](#frames): Frames hold reference frame information for the relative position of components in space.
+- [Webhooks](/build/configure/#webhooks): Webhooks allow you to trigger actions when certain types of data are sent from your machine to the cloud, or when the internet connectivity of your machine changes.
+- [Network](/build/configure/network): Networking options allow you to configure teh bind address for accepting connections.
 
 To start configuring, go to the [Viam app](https://app.viam.com) and create a new machine.
 Open the part status dropdown menu in the top left corner of the page, next to the machine's name.
 Click **View setup instructions** to open the setup instructions.
 Follow the appropriate instructions for your machine's architecture.
 
-The setup steps copy your machine's credentials to your machine and store them at <file>/etc/viam.json</file>.
-The credentials look like this:
-
-```json
-{
-  "cloud": {
-    "app_address": "https://app.viam.com:123",
-    "id": "123456ab-a1b2-12a3-a123-1a2b345cdefg6",
-    "secret": "LOCATION SECRET FROM THE VIAM APP"
-  }
-}
-```
-
-When you turn on your machine, `viam-server` starts up and, by default, it uses the credentials in its local <file>/etc/viam.json</file> to fetch its full config from the [Viam app](https://app.viam.com).
+The setup steps copy your machine's credentials to your machine.
+When you turn on your machine, `viam-server` starts up and uses the provided credentials to fetch its full config from the [Viam app](https://app.viam.com).
 Once the machine has a configuration, it caches it locally and can use the configuration for up to 60 days.
 The machine checks for new configurations every 15 seconds and changes its configuration automatically when a new configuration is available.
 All communication happens securely over HTTPS using secret tokens that are in a machine's configuration.
 
 {{< alert title="Tip" color="tip" >}}
+On Linux, the configuration is stored at <FILE>/etc/viam.json</FILE> by default and `viam-server` uses this configuration if no configuration is provided on startup.
+
 You can also store your config file in a custom location if desired. See [Run `viam-server`](/get-started/installation/manage/#run-viam-server) for more information.
```

Comment:
```suggestion
You can store your config file in a custom location if desired.
See [Run `viam-server`](/get-started/installation/manage/#run-viam-server) for more information.
```
Since you added the bit above, "also" should come out I think